cl/opinion_page/templates/opinion.html

```diff
@@ -212,6 +211,20 @@ <h3><span>Visualizations</span></h3>
 
         {% include "includes/social_links.html" %}
         {% include "includes/donate_sidebar.html" with referrer="o-donate-now" %}
+
+        {# Sponsored by #}
+        {% if sponsored %}
+            <div id="sponsored-by" class="sidebar-section">
+                <h3>
+                  <span>Sponsored By</span>
+                </h3>
+                <a href="https://www.vlex.com" rel="sponsored">
+                    <img id="vlex" src="{% static "img/vlex-white.png" %}" class="img-responsive v-offset-below-1" width="150" height="75" >
+                </a>
```

Comment:
Don't forget an alt text here and let's do a `target="_blank"` too.

Reply:
ah yes.